libraries/net/include/graphene/net/peer_connection.hpp

```diff
@@ -258,17 +258,17 @@ namespace graphene { namespace net
       // blockchain catch up
       fc::time_point transaction_fetching_inhibited_until;
 
-      uint32_t last_known_fork_block_number;
+      uint32_t last_known_fork_block_number = 0;
 
       fc::future<void> accept_or_connect_task_done;
 
-      firewall_check_state_data *firewall_check_state;
+      firewall_check_state_data *firewall_check_state = nullptr;
 #ifndef NDEBUG
     private:
-      fc::thread* _thread;
-      unsigned _send_message_queue_tasks_running; // temporary debugging
+      fc::thread* _thread = nullptr;
+      unsigned _send_message_queue_tasks_running = 0; // temporary debugging
 #endif
-      bool _currently_handling_message; // true while we're in the middle of handling a message from the remote system
+      bool _currently_handling_message = false; // true while we're in the middle of handling a message from the remote system
```

Comment:
This is public in release build, but private in debug build. I think it's better to make it consistent.

Reply:
done